App/Targets/three-dollar-in-my-pocket/Sources/domain/membership/signin-anonymous/SigninAnonymousView.swift

```diff
@@ -1,73 +1,40 @@
 import UIKit
 import AuthenticationServices
 
+import DesignSystem
 import Then
-import Lottie
 
 final class SigninAnonymousView: BaseView {
     let closeButton = UIButton().then {
         $0.setImage(UIImage(named: "ic_close_white"), for: .normal)
         $0.tintColor = .white
     }
     
-    let lottie = LottieAnimationView(name: "signin").then {
-        $0.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+    let logoText = UIImageView().then {
+        $0.image = UIImage(named: "img_signin_background")
         $0.contentMode = .scaleAspectFill
-        $0.loopMode = .loop
-        $0.play()
-        $0.clipsToBounds = false
     }
     
-    let kakaoButton = UIButton().then {
-        $0.layer.cornerRadius = 20
-        $0.backgroundColor = Color.kakaoYellow
-        $0.accessibilityLabel = "sign_in_with_kakao".localized
-    }
-    
-    private let kakaoLabel = UILabel().then {
-        $0.text = "sign_in_with_kakao".localized
-        $0.font = .bold(size: 14)
-        $0.textColor = UIColor.init(r: 56, g: 30, b: 31)
-        $0.isAccessibilityElement = false
-    }
-    
-    let kakaoImage = UIImageView().then {
-        $0.image = UIImage(named: "ic_kakao")
-    }
-    
-    let appleButton = ASAuthorizationAppleIDButton(type: .signIn, style: .white).then {
-        $0.cornerRadius = 24
-    }
-    
-    private let bottomContainerView = UIView().then {
-        $0.layer.cornerRadius = 20
-        $0.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
-        $0.backgroundColor = Color.gray90
-    }
+    let kakaoButton = SocialSigninButton(socialType: .kakao)
     
-    private let bottomImageView = UIImageView().then {
-        $0.image = UIImage(named: "img_anonymous")
-    }
+    let appleButton = SocialSigninButton(socialType: .apple)
     
     private let anonymousLabel = UILabel().then {
-        $0.font = .regular(size: 16)
+        $0.font = .regular(size: 14)
```

Comment:
이번에 v4올라가면서 폰트도 변경되어서 아래처럼 디자인시스템에서 가져다가 쓰시면 됩니다!!
```suggestion
        $0.font = DesignSystemFontFamily.Pretendard.regular.font(size: 14)
```

Reply:
헉 넵 알겠습니다 ! font도 들어온지 몰랐네요 수정하겠습니다 !